spec/spec.tex

```diff
@@ -1024,6 +1024,51 @@ \subsubsection{Format Strings}
 \item \verb|\'| : Single quote
 \end{itemize}
 
+\subsection{Verification}
+
+To facilitate simulation, model checking and formal methods, there are three non-synthesizable verification statements available: assert, assume and cover. Backends are free to generate the corresponding model checking constructs in the target language, but this is not required by the FIRRTL specification. For example, the System Verilog emitter produces System Verilog assert, assume and cover statements. However, backends should not generate any new synthesizable hardware when handling to verification statements. Backends that do not obey this rule should issue a warning.
```

Comment:
```suggestion
To facilitate simulation, model checking and formal methods, there are three non-synthesizable verification statements available: assert, assume and cover. Backends are free to generate the corresponding model checking constructs in the target language, but this is not required by the FIRRTL specification. For example, the SystemVerilog emitter produces SystemVerilog assert, assume and cover statements. However, backends should not generate any new synthesizable hardware when handling to verification statements. Backends that do not obey this rule should issue a warning.
```

Reply:
This changes `System Verilog` to `SystemVerilog` (👍)  in case anyone else is wondering...

Reply:
Hah, yes, the diff is not that clear.

Reply:
Fixed in 22876578ddfc1246f9341279f78b242397b79a06